test/test_constraint.jl

```diff
@@ -1597,6 +1620,9 @@ function test_semiinteger()
 end
 
 function test_symmetric_vectorize_allocations()
+    if VERSION < v"1.8"
+        return
+    end
```

Comment:
Intended?

Reply:
Yes, this allocated more for Julia v1.6 but couldn't figure out why.